DataFormats/L1THGCal/interface/HGCalClusterT.h

```diff
@@ -134,6 +134,46 @@ namespace l1t
       uint32_t subdetId() const {return detId_.subdetId();} 
       uint32_t layer() const {return detId_.layer();}
       int32_t zside() const {return detId_.zside();}
+
+      int32_t ClusterET()             const {return ClusterET_;}
```

Comment:
I see.
The issue I think is that the concept of 2D/3D clusters is orthogonal to Clusters/MultiClusters.  Indeed we could have the 2D/3D data formats available for both Clusters and MultiClusters. So maybe we can do something like that:
* Put all the 2D + 3D data format variables in `HGCalClusterT` (so remove the variables from the `MultiCluster` class)
* Change the naming of the variables to `cluster2D` and `cluster3D` instead of `cluster` and `multiCluster`, to make clear that these are different concepts. Actually it is not correct in the Twiki to write "MultiCluster", because our 3D clusters are not necessarily multi-clusters (it's only the case with this particular trigger architecture)
* Add an `enum` variable storing the type of cluster (2D or 3D, for the moment, and it can be extended if necessary), which you can access when coding/decoding.